src/container/tests/extensionApiSpec.js

```diff
@@ -2,18 +2,31 @@
 /* eslint-env mocha */
 
 import sinon from 'sinon';
+import _ from 'lodash';
 import { containerDoubles } from '../../../test/doubles';
 import { ArgError } from '../errors';
 import ExtensionApi from '../ExtensionApi';
 
 describe('ExtensionApi', () => {
+    let container;
+    let serviceId;
+    let serviceDefinition;
+    let extensionApi;
+
+    beforeEach(() => {
+        container = containerDoubles.container();
+        serviceId = 'foo';
+        serviceDefinition = {};
+        extensionApi = new ExtensionApi(container, serviceId, serviceDefinition);
+    });
 
     it('exposes a container with an extended chain', () => {
+        extensionApi.container.chain.should.deep.equal([serviceId]);
+    });
 
-        const extensionApi = new ExtensionApi({ chain: [] }, 'foo', {});
-
-        extensionApi.container.chain.should.deep.equal(['foo']);
-
+    it('exposes a the methods of the container', () => {
+        _.isUndefined(extensionApi.container.get).should.be.false;
```

Comment:
The more common approach I've seen is to expose `const should = chai.should();` and do `should.not.exist(myVal);` (I think it's recommended in chai docs). Not sure which I prefer. Although in this case `extensionApi.container.should.have.property('get');` is a lot nicer IMO 😄